gamestonk_terminal/forex/fx_controller.py

```diff
@@ -0,0 +1,177 @@
+import argparse
+from typing import List
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.forex import fx_view
+from gamestonk_terminal import config_terminal as cfg
+
+account = cfg.OANDA_ACCOUNT
+
+
+class ForexController:
+    """Oanda Controller class"""
+
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "price",
+        "summary",
+        "list",
+        "orderbook",
+        "positionbook",
+        "order",
+        "load",
+        "cancel",
+        "positions",
+        "closetrade",
+        "trades",
+        "candles",
+        "pending",
+        "calendar",
+    ]
+
+    def __init__(self):
+        """Construct Data"""
+        self.fx_parser = argparse.ArgumentParser(add_help=False, prog="fx")
+        self.fx_parser.add_argument(
+            "cmd",
+            choices=self.CHOICES,
+        )
+        self.instrument = None
+
+    @staticmethod
+    def print_help():
+        """Print help"""
+
+        print("\nForex Mode:")
+        print("    help          Show this menu again")
+        print("    q             Quit this menu and goes back to main menu")
+        print("    quit          Quit to abandon program")
+        print("")
+        print("    cancel        Cancel a pending order by ID -i order ID")
+        print("    candles       Show candles")
```

Comment:
I get this with `candles`:

```
(✨) (fx)> candles
{"errorMessage":"Invalid value specified for 'instrument'"}
```

and this is because I haven't loaded an instrument. So I think you should hide the commands that require an instrument. Similarly to how the main menu is working now, where it doesn't show Technical Analysis for instance because user hasn't load ticker.